src/cfnlint/decode/cfn_yaml.py

```diff
@@ -233,7 +233,7 @@ def load(filename):
         for line in fileinput.input(files=filename):
             content = content + line
     else:
-        with open(filename) as fp:
+        with open(filename, encoding='utf-8') as fp:
```

Comment:
Should this be done in more places as well?
https://sourcegraph.com/search?q=context:global+r:aws-cloudformation/cfn-lint+with+open(+-file:test/|scripts/
If so, wonder if there's any way to set a package-level default rather than hardcoding everywhere

Reply:
> Should this be done in more places as well?

In testing, yes.

> If so, wonder if there's any way to set a package-level default rather than hardcoding everywhere

I found a enforcing utf-8 codec option (PEP 540).
https://peps.python.org/pep-0540/
But I'm not familiar with setuptools to combine with `-x utf8` option and it's valid after python 3.7.